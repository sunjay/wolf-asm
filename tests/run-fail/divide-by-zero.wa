# This program divides a number by zero and triggers an exception at runtime

section .code

mov $1, 10
mov $2, 0x35

sub $2, $1
sub $2, $2

divu $1, $2
