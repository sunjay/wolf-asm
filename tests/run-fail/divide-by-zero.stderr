Error: Failed to execute instruction at `0x20`

Caused by:
    Divided a number by zero
